cpp/log/log_verifier.h

```diff
@@ -47,7 +48,7 @@ class LogVerifier {
       case INVALID_MERKLE_PATH:
         return "Invalid Merkle path.";
       default:
-        assert(false);
+        CHECK(false);
```

Comment:
You say the following ```return``` will now execute? It's *never* going to execute, you mean?

I'll add something to the ```CHECK```, though, good call.